docs/internationalization/core.md

```diff
@@ -214,10 +230,6 @@ If your integration provides entities under its domain, you will want to transla
 
 #### Entity state attributes
 
-:::info
-Translation of entity state attributes also requires frontend support, which is currently only available for `climate` entities.
-:::
-
```

Comment:
OK to remove this, or are there still limitations?

Reply:
I think we should remove it. There are still a few places left, but I rather see them reported as not working (so they can just be fixed). Most are there, especially all custom attributes.